addons/im_livechat/static/src/embed/common/thread_model_patch.js

```diff
@@ -64,4 +64,11 @@ patch(Thread.prototype, {
         this.store.env.services["im_livechat.chatbot"].bus.trigger("MESSAGE_POST", message);
         return message;
     },
+
+    get showUnreadBanner() {
+        if (this.chatbot && !this.chatbot.currentStep?.operatorFound) {
```

Comment:
Could be simpler in `master`, e.g. `chatbot` being falsy when operator takes over, or condition is on `correspondent` being chat bot (and when operator takes over, this ofc changes)